src/main/java/pl/futurecollars/invoicing/config/InvoiceConfiguration.java

```diff
@@ -23,21 +26,25 @@ public class InvoiceConfiguration {
 
   @Bean
   public JsonService jsonService() {
+    log.info("Creating jsonService");
     return new JsonService();
   }
 
   @Bean
```

Comment:
A nie chodziło o to, aby jsonService i fileService miał linijke @ConditionalOnProperty(name = "database.type", havingValue = "file")